src/Controllers/ShareDraftController.php

```diff
@@ -157,7 +157,9 @@ private function getRenderedPageByURL(string $url): HTTPResponse
         $variables = HTTPRequestBuilder::cleanEnvironment(Environment::getVariables());
         $variables['_SERVER']['REQUEST_URI'] = $url;
         $variables['_SERVER']['REQUEST_METHOD'] = 'GET';
-        $variables['_SERVER']['HTTP_USER_AGENT'] = 'CLI';
+        $variables['_SERVER']['HTTP_USER_AGENT'] = $variables['_SERVER']['HTTP_USER_AGENT']
+            ? $variables['_SERVER']['HTTP_USER_AGENT']
+            : 'CLI';
```

Comment:
```suggestion
        $variables['_SERVER']['HTTP_USER_AGENT'] =
                isset(variables['_SERVER']['HTTP_USER_AGENT']) &&
                $variables['_SERVER']['HTTP_USER_AGENT']
                ? $variables['_SERVER']['HTTP_USER_AGENT']
                : 'CLI';
```
Just wondering if we need an extra `isset()` check in here - or is the HTTP_USER_AGENT key always present, including from CLI?

Reply:
How I can see here https://www.php.net/manual/en/reserved.variables.server.php if user uses browser then browser should sends this header and we should get this value in _SERVER variable. But I can add isset(), just in case, to avoid unexpected warnings.